### PR TITLE
Data request CMS

### PR DESCRIPTION
**Pull request by Allcharles**
# Updated Data Request Page

Extracted boilerplate descriptions from Data Request page into CMS files.

## Changes

- Extracted download annotations description to CMS
- Added download annotations form

## Visual Changes

image

### CODE REVIEW

File: src/app/component/data-request/data-request.component.ts [hunk 1]
```diff
@@ -16,53 +17,62 @@ import { dataRequestCategory, dataRequestMenuItem } from "./data-request.menus";
...
   ngOnInit() {
-    this.loading = false;
+    this.requestLoading = false;
+  }
+
+  /**
+   * Form submission
+   * @param $event Form response
+   */
+  submitAnnotation($event: any) {
```
Comment (atruskie) [inline src/app/component/data-request/data-request.component.ts]: why is this named annotation?

Comment (atruskie) [reply src/app/component/data-request/data-request.component.ts]: Oh I see. Maybe "submitDownloadAnnotations"

**Review by atruskie — APPROVED** [review]
Why are the forms so skinny?

**Comment by Allcharles** [comment]
All forms currently maintain the same size, we can create a toggle between large and small forms if you want them larger.
@atruskie

**Comment by atruskie** [comment]
> All forms currently maintain the same size, we can create a toggle between large and small forms if you want them larger.
> @atruskie

I think they should fill to 100% of their container by default. We can use CSS to override container width where needed afterwards.

**Review by atruskie — APPROVED** [review]

**Pull request merged**